core/src/main/scala/org/apache/spark/util/collection/PartitionedPairBuffer.scala

```diff
@@ -90,3 +99,7 @@ private[spark] class PartitionedPairBuffer[K, V](initialCapacity: Int = 64)
     }
   }
 }
+
+private[spark] object PartitionedPairBuffer {
```

Comment:
This could actually be strictly private. Not a big deal at all if you decide to keep this as is.